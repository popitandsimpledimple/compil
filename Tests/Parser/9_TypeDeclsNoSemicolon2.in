program main;
type int = integer MyDouble = double
begin
end.