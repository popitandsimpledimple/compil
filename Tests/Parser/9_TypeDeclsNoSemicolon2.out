(2,20) Expected 'Semicolom' Found 'MyDouble'